internal/cypress/config.go

```diff
@@ -19,6 +20,7 @@ type Project struct {
 	config.TypeDef `yaml:",inline"`
 	Defaults       config.Defaults `yaml:"defaults" json:"defaults"`
 	ShowConsoleLog bool
+	RawConfig      string             `yaml:"-" json:"-"`
```

Comment:
I think it might be more useful further down the line if this was a path to the config, rather than its raw contents. Your current approach works just fine, so just food for thought!

Reply:
@alexplischke I did it this way to avoid re-opening/reading the file.